Sc 30754 Create Quantum Computing hub (#684)
PennyLaneAI/qml: quantum-computing.rst

.. raw:: html

    <style>
        h1 {
            text-align: center;
            width: auto;
            margin: 40px 0 0 0;
            padding: 0;
        }
        h2 {
            width: 65%;
        }
        .info-card {
            min-width: 260px;
            margin: 0 10px 0 10px;
            border-radius: 10px
        }
        .card-footer {
            background-color: white;
            border: none;
        }
        .card-body {
            padding-top: 10px;
        }
        .video .card {
            max-width: 600px;
            border-radius: 10px
        }
        .video-card-text {
            font-size: 24px;
            margin: 0;
            padding: 15px;
            border: none;
            width: 100%;30754
        }
        .mt-50 {
            margin-top: 50px;
        }
    </style>

> Quantum Computing learning hub
> ==============================

Reviewer: ```suggestion
Quantum Computing
=================
```

.. meta::

>    :property="og:description": Placeholder until copy is provided

Reviewer: ```suggestion
   :property="og:description": Sit back and learn about quantum computing. Explore key concepts, download and run demonstrations, and view our selection of curated videos.
```

   :property="og:image": https://pennylane.ai/qml/_static/qml_card.png

.. raw:: html

    <link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.5.14/css/mdb.min.css" rel="stylesheet">
    <div>
        <p class="lead grey-text text-center mx-auto my-5">

>         Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Curabitur tempus urna at turpis condimentum lobortis.

Reviewer: ```suggestion
        Quantum computing is a research area that extends the set of physical laws classical computers operate on by accessing <strong>quantum aspects of the physical world</strong>, opening up new ways of processing information.
        <br>
        As a highly interdisciplinary field of quantum science and technology, quantum computing sits at the meeting point of algorithm design, complexity theory, system architecture, and hardware development.
```

Reviewer: @ikurecic I copied some of your what is content here, feel free to suggest modifications!

Reviewer 2: @josh146 — Ana and I were hoping to get approval on the final revisions before copying in the text, so we don't end up doing the work twice. I'm sorry we didn't communicate that with you directly. I'll definitely put in the final copy once that's sorted.

Reviewer: No worries! I'll leave this here as a comment

        </p>

        <section class="my-5">
            <div class="row main-cards d-flex justify-content-center">
                <div class="info-card col-lg-3 mb-5">
                    <a href="what-is-quantum-computing.html">
                        <div class="card rounded-lg h-100">
                        <div class="card-body text-center">
                            <h3 class="card-title">
                                <img src="_static/brain_board.png" class="img-fluid mb-2" style="max-width: 86px;" alt="brain board"></img>
                                <br>
                                What is Quantum Computing?
                            </h3>
                            <p class="card-text">

>                             Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the

Reviewer: ```suggestion
                            Understand what quantum computers can do and how we can make them do it.
```

                            </p>
                        </div>
                        <div class="card-footer">
                            <div class="white-text d-flex justify-content-center"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                        </div>
                        </div>
                    </a>
                </div>

                <div class="info-card col-lg-3 mb-5">
                    <a href="glossary.html">
                        <div class="card rounded-lg h-100">
                        <div class="card-body text-center">
                            <h3 class="card-title">
                                <img src="_static/key.png" class="img-fluid mb-2" style="max-width: 80px;" alt="key"></img>
                                <br>
                                Key Terms
                            </h3>
                            <p class="card-text">

>                             Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the

Reviewer: ```suggestion
                            Explore different concepts underpinning quantum computation.
```

@ikurecic @BM7878 this one could be reworked, the glossary page is a bit of a weird one 🤔

                            </p>
                        </div>
                        <div class="card-footer">
                            <div class="white-text d-flex justify-content-center"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                        </div>
                        </div>
                    </a>
                </div>
                
                <div class="info-card col-lg-3 mb-5">
                    <a href="demonstrations.html">
                        <div class="card rounded-lg h-100">
                        <div class="card-body text-center">
                            <h3 class="card-title">
                                <img src="_static/flask.png" class="img-fluid mb-2" style="max-width: 53px;" alt="blue flask"></img>
                                <br>
                                Demos
                            </h3>
                            <p class="card-text">

>                             Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Curabitur tempus urna at turpis condimentum lobortis.

Reviewer: ```suggestion
                            Take a dive into quantum computing and explore the physics, concepts, and algorithms through code.
```

                            </p>
                        </div>
                        <div class="card-footer">
                            <div class="white-text d-flex justify-content-center"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                        </div>
                        </div>
                    </a>
                </div> 
            </div>
        </section>

        <hr class="pb-4 text-dark">

        <section>
            <div class="d-flex flex-wrap justify-content-center">

>                 <h2 class="fw-normal text-center mt-1 mb-5 border-0">Curabitur tempus urna at turpis lobortis. Ut commodo efficitur neque.</h2>

Reviewer: ```suggestion
                <h2 class="fw-normal text-center mt-1 mb-5 border-0">Sit back and explore quantum computing with our curated selection of expert videos.</h2>
```

            padding: 0;
        }
        h2 {
            width: 65%;
        }
        .info-card {
            min-width: 260px;
            margin: 0 10px 0 10px;
            border-radius: 10px
        }
        .card-footer {
            background-color: white;
            border: none;
        }
        .card-body {
            padding-top: 10px;
        }
        .video .card {
            max-width: 600px;
            border-radius: 10px
        }
        .video-card-text {
            font-size: 24px;
            margin: 0;
            padding: 15px;
            border: none;
            width: 100%;30754
        }
        .mt-50 {
            margin-top: 50px;
        }
    </style>

Quantum Computing
=================

.. meta::
   :property="og:description": Sit back and learn about quantum computing. Explore key concepts, download and run demonstrations, and view our selection of curated videos.
   :property="og:image": https://pennylane.ai/qml/_static/qml_card.png

.. raw:: html

    <link href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.5.14/css/mdb.min.css" rel="stylesheet">
    <div>
        <p class="lead grey-text text-center mx-auto my-5">

>         Quantum computing is a research area that extends the set of physical laws classical computers operate on by accessing <strong>quantum aspects of the physical world</strong>, opening up new ways of processing information.

Reviewer 2: ```suggestion
        Quantum computing is a research area that extends the set of physical laws classical computers operate on by accessing quantum aspects of the physical world, opening up new ways of processing information.
```
We didn't use emphasis in other hub pages, so for consistency.

                        </div>
                    </a>
                </div>

                <div class="info-card col-lg-3 mb-5">
                    <a href="glossary.html">
                        <div class="card rounded-lg h-100">
                        <div class="card-body text-center">
                            <h3 class="card-title">
                                <img src="_static/hubs/key.svg" class="img-fluid mb-2" style="max-width: 130px;" alt="Blue key"></img>
                                <br>
                                Key Terms
                            </h3>
                            <p class="card-text">
                            Explore different concepts underpinning quantum computation.
                            </p>
                        </div>
                        <div class="card-footer">
                            <div class="white-text d-flex justify-content-center"><h5>Read more <i class="fas fa-angle-double-right"></i></h5></div>
                        </div>
                        </div>
                    </a>
                </div>
                
                <div class="info-card col-lg-3 mb-5">

>                     <a href="demonstrations.html">

Reviewer: ```suggestion
                    <a href="demos_quantum-computing.html">
```

Only if we make the same change as discussed in https://github.com/PennyLaneAI/qml/pull/682#discussion_r1092112760
